integrations/security/OCSF_1_1_0/readme.md

```diff
@@ -0,0 +1,100 @@
+# README
+# OCSF 1.1.0 integration with OpenSearch
+
+## Overview
+This section provides resources that will help you ingest [Open Cybersecurity Schema Framework](https://schema.ocsf.io/1.1.0/) (OCSF) logs into OpenSearch and use the logs for [Security Analytics](https://opensearch.org/docs/latest/security-analytics/)
+
+ It consists of index and component templates, OpenSearch Ingestion template, [Index State Management](https://opensearch.org/docs/latest/im-plugin/ism/policies/) policy initialization scripts, and saved objects (visualizations and index templates).
+
+## Overview of components
+### Index and component templates
+Index templates (`schemas/index_templates`) automatically apply predefined settings and mappings to indices. 
+
+Component templates (`schemas/component_templates`) are reusable building blocks that contain mapping definitions. Component templates are used as part of index templates. 
+
+The current set of index and component templates are mapped to the OSCF 1.1.0 standard. The repository contains index templates for the following OCSF 1.1.0 categories and classes:
```

Comment:
can you add a visual diagram of the different OCSF entities that are presented here ?
and also a link to the OCSF url resources?

Reply:
we can hotlink this to the spec for each class_uid....beware it could break if the OCSF project reorgs the structure of the presentation on the current site